TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/SelfDestructingMessageView.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.tpbot;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+/**
+ * A {@link TextView} that clears its content after a set period of time
+ */
+public class SelfDestructingMessageView extends TextView {
+
+    public SelfDestructingMessageView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public void showPermanently(String arrowCharacter) {
+        super.setText(arrowCharacter);
```

Comment:
Forgot to change them back after realizing that setText() was `final`. I was going to through an exception when that was called.

Done in 	5d56687